spec/features/user_views_recipes_spec.rb

```diff
@@ -1,14 +1,15 @@
 require 'rails_helper'
 
 feature 'User views recipes' do
-  let!(:recipe) { FactoryGirl.create(:recipe) }
-  let!(:recipe2) { FactoryGirl.create(:recipe, title: 'Another Recipe') }
+  let!(:user) { FactoryGirl.create(:user) }
+  let!(:recipe) { FactoryGirl.create(:recipe, user: user) }
+  let!(:recipe2) { FactoryGirl.create(:recipe, title: 'Another Recipe',\
```

Comment:
Fix this, I don't think you need to use the \, just make sure user: aligns with :recipe